app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+from typing import List
+
+
+class Animal:
+    alive: List["Animal"] = []
+
+    def __init__(self, name: str) -> None:
+        self.name: str = name
+        self.health: int = 100
+        self._hidden: bool = False
+        Animal.alive.append(self)
+
+    @property
+    def hidden(self) -> bool:
+        return self._hidden
```

Comment:
What is the purpose of creating `_hidden` and then accessing through `hidden`?)
I dont see in the task that we need private hidden attr